native/explorer/Cargo.toml

```diff
@@ -79,7 +81,7 @@ version = "0.31"
 [features]
 default = ["ndjson", "cloud", "nif_version_2_15"]
 
-cloud = ["aws"]
+cloud = ["object_store", "tokio", "tokio-util", "aws"]
 ndjson = ["polars/json"]
 aws = ["polars/async", "polars/aws"]
```

Comment:
Which way around do we want these features to depend?

Do we want the `cloud` feature to depend on all possible provider features (e.g. `aws`/`gcp`/`azure`/`...`), or have `cloud` be the 'base' feature and features like `aws` depend on it?

Polars itself does the former.